apps/deploy-web/tests/ui/fixture/Leap/async/4578.js

```
@@ -239,17 +254,17 @@ _global.SENTRY_RELEASE = { id: "0.22.9" };
         p = Object.getOwnPropertySymbols,
         x = Object.prototype.hasOwnProperty,
-        h = Object.prototype.propertyIsEnumerable,
-        f = (e, t, a) => (t in e ? d(e, t, { enumerable: !0, configurable: !0, writable: !0, value: a }) : (e[t] = a)),
-        g = (e, t) => {
-          for (var a in t || (t = {})) x.call(t, a) && f(e, a, t[a]);
-          if (p) for (var a of p(t)) h.call(t, a) && f(e, a, t[a]);
+        g = Object.prototype.propertyIsEnumerable,
+        h = (e, t, a) => (t in e ? d(e, t, { enumerable: !0, configurable: !0, writable: !0, value: a }) : (e[t] = a)),
+        f = (e, t) => {
+          for (var a in t || (t = {})) x.call(t, a) && h(e, a, t[a]);
+          if (p) for (var a of p(t)) g.call(t, a) && h(e, a, t[a]);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'a'. Consider to delete it or rename it.

'a' is defined here:


(lint/suspicious/noRedeclare)